### PR TITLE
Refer to the NeurIPS demo in README

### PR DESCRIPTION
**Pull request by borzunov**
Screenshot 2021-12-21 at 18 12 43

**Note:** The words "join an ongoing collaborative training" will be removed when the training is stopped.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ large model on hundreds of computers from different universities, companies, and
 
 ![img](https://i.imgur.com/GPxolxb.gif)
 
+## Live Demo
+
+Check out our NeurIPS 2021 demonstration ["Training Transformers Together"](https://training-transformers-together.github.io/) to see hivemind in action, join an ongoing collaborative training, and learn more about the technologies behind it!
```
Comment (mryab) [inline README.md:17]:
```suggestion
Check out our NeurIPS 2021 demonstration ["Training Transformers Together"](https://training-transformers-together.github.io/) to see hivemind in action, join an ongoing collaborative experiment, and learn more about the technologies behind it!
```

**Review by mryab — COMMENTED** [review]
Thanks for the PR! To clarify, are we going to place this website in the repo description?

**Comment by mryab** [comment]
Also, maybe when the experiment is stopped, we should put the library description back to the top and put the demo description inside "Key Features": basically, it's a showcase of a part of hivemind, so it seems like an appropriate location

**Comment by borzunov** [comment]
> To clarify, are we going to place this website in the repo description?

As you wish! I'd put the link there, however, I had the impression that you're against it.

As agreed, I've removed the old link. Also, I've made a related PR to the old webpage: https://github.com/learning-at-home/learning-at-home.github.io/pull/1

**Review by mryab — APPROVED** [review]
> I'd put the link there, however, I had the impression that you're against it (so now I don't plan to do that).

> As agreed, I've removed the old link.

Sure, I got no objections against that. I just didn't want the demo link to be _the link_ about hivemind (e.g. to be listed in the sidebar on the right), since it doesn't fully convey the scope of the library while telling the reader many other things which are less relevant

**Pull request merged**